flight/Modules/Stabilization/stabilization.c

```diff
@@ -372,6 +372,25 @@ static void stabilizationTask(void* parameters)
 		static uint8_t previous_mode[MAX_AXES] = {255,255,255};
 		bool error = false;
 
+		// Re-project axes if necessary prior to running stabilization algorithms.
+		uint8_t reprojection = stabDesired.ReprojectionMode;
+		if (reprojection == STABILIZATIONDESIRED_REPROJECTIONMODE_CAMERAANGLE) {
+				float camera_tilt_angle = settings.CameraTilt; // XXX(fujin): any reason to assign this? readability?
+				if (camera_tilt_angle) {
+						float roll = stabDesired.Roll;
+						float yaw = stabDesired.Yaw;
+						float halfpi = (float)(M_PI / 180.0);
+						// The roll input should be equal to the cosine of half of pi times the camera angle times the roll,
+						// added to the sine of half pi times the camera angle times the yaw.
+						stabDesired.Roll = (cosf(halfpi * camera_tilt_angle) * roll +
+																(sinf(halfpi * camera_tilt_angle) * yaw));
+						// Yaw is similar but uses the negative sine of half pi times the camera angle, times the roll,
```

Comment:
remove the 'half pi' here, it's not the value (it's 1/180th of pi) .. it's just the negative sine of the camera angle.  Just we store the angle in degrees and the math libraries want radians